Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -93,20 +100,77 @@ namespace Terrain
-                        ->Attribute(AZ::Edit::Attributes::Max, 16.0f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMax, 128.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 100.0f)
                     ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution,
-                        "Detail clipmap max resolution (in texels per meter)",
+                        "Detail clipmap max resolution: texels/m",
                         "The resolution of the highest resolution clipmap in the stack.")
-                        ->Attribute(AZ::Edit::Attributes::Min, 0.5f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMin, 256.0f)
-                        ->Attribute(AZ::Edit::Attributes::Max, 2048.0f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4096.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2048.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4096.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapScaleBase,
+                        "Macro clipmap scale base",
+                        "The scale base between two adjacent clipmap layers. \n"
+                        "For example, 3 means the (n+1)th clipmap covers 3^2 = 9 times to what is covered by the nth clipmap.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapScaleBase,
+                        "Detail clipmap scale base",
+                        "The scale base between two adjacent clipmap layers. \n"
+                        "For example, 3 means the (n+1)th clipmap covers 3^2 = 9 times to what is covered by the nth clipmap.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMarginSize,
+                        "Macro clipmap margin size: texels",
+                        "The margin of the clipmap where the data won't be used, so that bigger margin results in less frequent update."
+                        "But bigger margin also means it tends to use lower resolution clipmap.")
```

Comment:
Suggestion:
"The margin of the clipmap beyond the visible data. Increasing the margins results in less frequent clipmap updates but also results in lower resolution clipmaps rendering closer to the camera."